assignments/objects.js

```diff
@@ -49,16 +106,31 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": 70,
+  speak: function() {
+    return "My name is " + this.name + "!";
+   },
+  child: {
+    "name": "George",
+    "age": 50,
```

Comment:
adding the `speak` function here the same way it's written on 112 would work for `parent.child.speak()`.  Thus the power of `this`.